### PR TITLE
changed baseUrls.image from s to zed

### PR DESCRIPTION
**Pull request by nanetteCodes**
issue: https://github.com/triniti/cms-js/issues/17

### CODE REVIEW

File: src/plugins/dam/utils/damUrl.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const tld = API_ENDPOINT.split('://').pop().split('/').shift()
 
 export const baseUrls = {
   dfault: `https://dam.${APP_ENV === 'prod' ? '' : `${APP_ENV}.`}${tld}/`,
-  image: `https://images${APP_ENV === 'prod' ? '.' : `-${APP_ENV}.`}${tld}/`,
+  image: `https://imagez${APP_ENV === 'prod' ? '.' : `-${APP_ENV}.`}${tld}/`,
```
Comment (albertgcsula) [inline src/plugins/dam/utils/damUrl.js]: @gdbrown due to the differences in the image base urls i.e. https://imagez-dev.toofab.com vs. https://bachelornation-images-dev.akamaized.net. Can we just set something like: IMAGE_BASE_URL=https://imagez-dev.toofab.com/ in the env file and use image:IMAGE_BASE_URL here?

Comment (gdbrown) [reply src/plugins/dam/utils/damUrl.js]: @albertgcsula not opposed to that, was trying to get them to be the same strategy as the symfony apps and let the index.js or config on the local site to customize it as needed.  either way.  if using globals then you have some more wiring to do in other places and you'd want to do DAM_BASE_URL and IMAGE_BASE_URL

**Comment by albertgcsula** [comment]
@efjacobson asking for your review on my changes, thanks.

**Comment by efjacobson** [comment]
looks good to me. @gdbrown i wasnt able to find any other places that need wiring

**Comment by gdbrown** [comment]
@efjacobson when you see it, you'll know.  (it's at the site level)

**Pull request merged**